routes/api/follows.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express');
+const router = express.Router();
+const createError = require('http-errors');
+const { sequelize, Follow } = require("../../db/models");
+const {asyncHandler} = require('../utils');
+
+/* Delete a follower from a user */
+router.delete('/follows/:id', asyncHandler(async (req, res, next) => {
+
+        /* Initialize follow object */
+        const follow = await Follow.findPk(req.params.id);
+
+        /* Return error 404, if 
+           there is no follow object */
+        if (typeof follow === "undefined" ) {
```

Comment:
A query with no result ends up being `null`. Best to just use `if (!follow)`

Reply:
Suggestion taken and implemented.